### DISCUSSION TITLE
how to edit files? winscp?

### DISCUSSION THREAD

**Discussion by vincom** [gtxaspec/wz_mini_hacks#238] [Q&A] [answered]
for v3. i got upto ssh working on putty, can log into camera , explore camera file structure with cd and ls commands.
cant get scp with winscp to work though.
 to change rtsp settings, do u edit the file on the sdcard while its in the pc then transfer  sdcard to camera and then boot camera . if so can u edit the file while its in the  camera with winscp 
where is wz_mini.conf located, i cant seem to find it when using putty, its not in the root. i can see it in the wz_mini folder  when sdcard is in the pc 
is there a way to disable ssh

edit. answered my first question. yes, edit the wz_mini.conf file b4 putting it on camera, got rtsp working in bi.
just need to know how to edit the files in winscp, the app i normally use to get into linux devices.

**EDIT #2** finally got winscp to work, was using an older version so i updated the app to 5.21.2, changed environment/scpshell/  to bin/sh, as posted in 
https://github.com/gtxaspec/wz_mini_hacks/discussions/135#discussioncomment-3043296

ive been using winscp for years but ive not used it recently. being on win pc its the best program to navigate and browse through the linux file system and edit files, imho

**Comment by gtxaspec** [comment]
you can use `nano` or `vi`, both are console based text editors

**Comment by vincom** [comment, marked as answer]
im on a win machine, but problem right now is i cant seem to find "wz_mini.conf" file using putty.

**Reply by baudneo** [reply]
`nano /opt/wz_mini/wz_mini.conf`

Make your changes then hit ctrl+s to save and then ctrl+x to exit nano.

Nano is a Linux terminal text editor 

To have the settings take effect type `reboot` to restart the camera.

**Reply by vincom** [reply]
> `nano /opt/wz_mini/wz_mini.conf`
> 
> Make your changes then hit ctrl+s to save and then ctrl+x to exit nano.
> 
> Nano is a Linux terminal text editor
> 
> To have the settings take effect type `reboot` to restart the camera.

sheesh, there it is, thx buddy. no more   swapping  sdcard between pc and camera

**Comment by Bob-AOL** [comment]
There's an editor included on the webserver. Hardly secure, but functional.

**Comment by vincom** [comment]
finally got winscp to work, was using an older version so i updated the app to 5.21.2, changed environment/scpshell/  to bin/sh, as posted in 
https://github.com/gtxaspec/wz_mini_hacks/discussions/135#discussioncomment-3043296.

ive been using winscp for years but ive not used it recently. being on win pc its the best program to navigate and browse through the linux file system and edit files, imho
